packages/cozy-harvest-lib/src/models/KonnectorJob.js

```diff
@@ -47,6 +48,7 @@ export class KonnectorJob {
     this.handleTwoFA = this.handleTwoFA.bind(this)
     this.launch = this.launch.bind(this)
     this.sendTwoFACode = this.sendTwoFACode.bind(this)
+    this.onUnmount = this.onUnmount.bind(this)
```

Comment:
KonnectorJob should not have any knowledge about React components lifecycle. So this method should not be named after a React concept like unmounting.

I suggest instead `unwatch()`.